utils.go

```diff
@@ -49,8 +49,12 @@ func toJsonArrayOfNum(s string) string {
 }
 
 func toJsonStr(r interface{}) (string, error) {
-	data, err := json.Marshal(r)
-	return string(data), err
+	var buf bytes.Buffer
+	enc := json.NewEncoder(&buf)
+	enc.SetEscapeHTML(false)
+	err := enc.Encode(r)
+
+	return strings.TrimSpace(buf.String()), err
 }
```

Comment:
According to the docs here https://golang.org/pkg/encoding/json/#Encoder.Encode you get a bonus newline character. So if we are concerned that trimming white space might change the actual content, maybe we could use this:

strings.TrimSuffix(buf.String(),"\n")

The tests still pass.